thirdParty/cupla/.gitmodules

```diff
@@ -1,4 +0,0 @@
-[submodule "alpaka"]
-	path = alpaka
-	url = git://github.com/ComputationalRadiationPhysics/alpaka.git
```

Comment:
@psychocoderHPC you might also want to consider adding this url to a https:// url on cupla itself.
Otherwise, anonymous git clones will fail for cupla.